packages/disploy/README.md

```diff
@@ -17,56 +17,177 @@
 
 </div>
 
-> **Warning**: We're still in development, and the packages published are not up to date. They're just early builds that we've published to reserve our package names on NPM.
+> **Warning**: We're still in development, and packages are published to NPM every 12 hours to the `@dev` tag.
 
-Disploy is a framework for building Discord bots with ease. It's designed to make it easy to build, test and deploy Discord bots.
+Disploy is a library for building HTTP interaction-based Discord bots with ease. It's designed to make it easy to build, test and deploy Discord bots.
 
-## It's as easy as 1, 2, 3, 4
+# Features
 
-### #1
+Disploy features a [library](#library) and an opinionated [framework](#framework) with tooling inspired by Next.js.
+
+## Library
+
+> Disploy does not come included with a "server", that's up to you to implement. We have a [guide](https://disploy.dev/docs/Reference/framework-less/) showcasing you how to do so with Express (Node.js) and Deno's inbuilt server.
+
+In this readme, we have supplied you with a more slimmed-down guide on usage with Next.js as your server.
+
+### Usage with Next.js
+
+The API entry point
+
+```ts
+// Entrypoint - pages/api/interactions.ts
+import { createNextAdapter } from 'disploy';
+import { App } from '../../lib/main';
+
+export default createNextAdapter(SafecordApp);
```

Comment:
Where's the import for SafecordApp? Do you mean `App`? If so,
```suggestion
export default createNextAdapter(App);
```

Reply:
Maybe ExampleApp is a better name?

Reply:
Sure

Reply:
resolved in https://github.com/Disploy/disploy/pull/68/commits/2c16218f008efe8d1d36b7b80fbe5b4e6fcbfbd8